src/components/ui/form/TextInput/TextInput.jsx

```diff
@@ -0,0 +1,18 @@
+import PropTypes from "prop-types";
+import "./TextInput.scss";
+
+function TextInput({ className, type, ...rest }) {
```

Comment:
It may confuse with `TextArea`. We can just say it `Input`

Reply:
I've changed it to `Input`